README.md

```diff
@@ -1,13 +1,88 @@
-# ReactiveLists
+![](Resources/logo.png)
+
 
 *React-like API for `UITableView` & `UICollectionView`*
 
 [![Build Status](https://travis-ci.org/plangrid/ReactiveLists.svg?branch=master)](https://travis-ci.org/plangrid/ReactiveLists) [![Version Status](https://img.shields.io/cocoapods/v/ReactiveLists.svg)][podLink] [![license MIT](https://img.shields.io/cocoapods/l/ReactiveLists.svg)][mitLink] [![codecov](https://codecov.io/gh/plangrid/ReactiveLists/branch/master/graph/badge.svg)](https://codecov.io/gh/plangrid/ReactiveLists) [![Platform](https://img.shields.io/cocoapods/p/ReactiveLists.svg)][docsLink]
 
-## About
+`ReactiveLists` provides a React-like API for `UITableView` and `UICollectionView` that makes it easy to write stateless code that generates user interfaces.
+
+Instead of spreading the definition of your content over various data source methods, you can define the content as concisely as in the real-world example below.
+
+## Features
+
+- React-like declarative API for `UITableView` and `UICollectionView`
+- Automatic UI updates, whenever your models change
+
+## Real-world example
```

Comment:
How do you feel about putting this in the "Getting Started" guide instead?

Reply:
I could see it going either way.  Its cool that we show off a real implementation of the code from our code base, but I guess its technically more complicated that it needs to be for something like `Getting Started`?  Not too complicated, but I guess it does have several `*ViewModels`